forge-game/src/main/java/forge/game/ability/effects/SetStateEffect.java

```diff
@@ -196,4 +204,34 @@ public void resolve(final SpellAbility sa) {
             game.getAction().reveal(transformedCards, p, true, "Transformed cards in ");
         }
     }
+
+    private static void setFaceDownState(Card c, SpellAbility sa) {
```

Comment:
Can you just move the one from ChangeZoneEffect to a common place?

Reply:
Absolutely!